client/src/jobs/job.py

```diff
@@ -416,9 +422,7 @@ def _render_dockerfile(self) -> str:
             raise ValueError("Container image spec must be specified")
 
         if not image_spec.is_file():
-            raise FileNotFoundError(
-                f"Container image spec file not found: {image_spec.is_file()}"
-            )
+            raise FileNotFoundError("Container image spec file not found")
```

Comment:
`FileNotFoundError` accepts the filename as a second argument after the error message, so we should set it (see docs for [`OSError`](https://docs.python.org/3/library/exceptions.html#OSError), which it inherits from)